### PR TITLE
chore: use SendMsg for send_node_msg_to_nodes

### PR DESCRIPTION
**Pull request by joshuef**
As it always results in a delivery group of 1. Which is pointless

### CODE REVIEW

File: sn_node/src/node/data/records/mod.rs [hunk 1]
```diff
@@ -215,40 +214,42 @@ impl Node {
...
-        let close_full_adults = if let Some(closest_empty) = candidates.iter().next() {
+        let candidates_clone = candidates.clone();
+        let close_full_adults = if let Some(closest_empty) = candidates_clone.iter().next() {
             full_adults
                 .iter()
                 .filter_map(|name| {
-                    if target.cmp_distance(name, closest_empty) == Ordering::Less {
-                        Some(*name)
+                    if target.cmp_distance(name, &closest_empty.name()) == Ordering::Less {
+                        Some(closest_empty)
```
Comment (Yoga07) [inline sn_node/src/node/data/records/mod.rs:223]: Was this wrong before? :thinking:

Comment (joshuef) [reply sn_node/src/node/data/records/mod.rs:223]: ah no, i've borked that. Good spot!

Comment (joshuef) [reply sn_node/src/node/data/records/mod.rs]: sorted

**Review by Yoga07 — APPROVED** [review]

File: sn_node/src/node/api.rs [hunk 2]
```diff
@@ -130,7 +130,7 @@ impl Node {
     }
 
     // Send message to peers on the network.
-    pub(crate) fn send_msg_to_nodes(&self, mut wire_msg: WireMsg) -> Result<Option<Cmd>> {
+    pub(crate) fn send_msg_on_to_nodes(&self, mut wire_msg: WireMsg) -> Result<Option<Cmd>> {
```
[the remark this answers is not in the document]
Comment (joshuef) [reply sn_node/src/node/api.rs:133]: yeh, that was my thinking, as it talks about `relay` below. Happy for other ideas (or i can revert the name change... but i also wanted to distinguish this from other funcs of similar name)

[the remark this answers is not in the document]
Comment (oetyng) [reply sn_node/src/node/api.rs:133]: Ah, yeah not too important, was just wondering. Might have some naming idea eventually. These send msg fns with similar names are moving towards less confusion bit by bit :)

**Pull request merged**